releasenotes/notes/k8s-file-scanning-improvements-e9a602c1fe7b0461.yaml

```diff
@@ -11,3 +11,5 @@ enhancements:
     Kubernetes pod with short-lived containers won't have a few logs of lines
     duplicated with both container tag (the stopped one and the running one) anymore
     while logs are being collected.
+    Mount `/var/log/containers` and use `logs_config.validate_pod_container_id`
```

Comment:
reST formatting:

```suggestion
    Mount ``/var/log/containers`` and use ``logs_config.validate_pod_container_id``
```

Reply:
Arr I always forgot about that, thanks 🙇